reliability/config/enhance_reliability.yaml

```diff
@@ -1,15 +1,13 @@
 reliability:
   kubeconfig: ~/Downloads/kubeconfig
   timeSubstitutions:
-    minute: 10s
-    hour: 30s
+    minute: 30s
+    hour: 1m
     day: 2m
-    week: 3m
-    month: 4m
+    week: 6m
+    month: 6m
   limits:
-    maxProjects: 20
-    # sleepTime should be <= mix of timeSubstitutions in the current design,
-    # or else, tasks with timeSubstitutions < sleepTime will be executed with the interval of sleepTime.
+    maxProjects: 25
```

Comment:
It looks like with concurrency this has become a "projects per user number".  I am seeing 100+ projects active after a while which is more than a "standard" 3 worker cluster can handle.   We should change this number to 5 or 6 or else make it a global max.   Either is fine with me as long as we document it with a comment.

Reply:
Yes, this is a per project now. Let me change the default number and then put some description to it, about how to set a good number considering the concurrency and the capacity of the cluster.

Reply:
Jenkins is not working today, I used a manual installed aws cluster which does not have the 50 test users. So I didn't test multiple concurrent users yet. I added some debug output and did a test with admin user - concurrency=1, I can get the warning 
```
current_projects:25
max_projects:25
2021-06-16 19:58:18,830 - WARNING - create_project: project kubeadmin-796. 25 already exist.
```
I will test with real multiple users later.
I guess there are script to create the 50 test users for a manually installed cluster but haven't get chance to fine it, I'll figure out. Let me know if you know where it is.